src/BuildScriptGenerator/Python/JupyterNotebookBashBuildSnippet.sh.tpl

```diff
@@ -6,7 +6,7 @@ conda info
 {{ if EnvironmentYmlFile | IsNotBlank }}
 envFile="{{ EnvironmentYmlFile }}"
 {{ else }}
-envFile="oryx.environment.yml"
+envFile="/opt/oryx/conda/oryx.environment.yml"
```

Comment:
we were creating this file in user's source repo location which needs to be avoided